Tests/TuistCoreTests/Xcode/XcodeControllerTests.swift

```diff
@@ -47,4 +48,31 @@ final class XcodeControllerTests: XCTestCase {
         // Then
         XCTAssertNotNil(xcode)
     }
+
+    func test_selectedVersion_when_xcodeSelectDoesntReturnThePath() throws {
+        // Given
+        system.errorCommand(["xcode-select", "-p"])
+
+        // Then
+        XCTAssertThrowsError(try subject.selectedVersion())
```

Comment:
I'd assert that the error that is being thrown is the expected one.

Reply:
That's probably better, added.